source/uwp/Renderer/dll/CppWinRTIncludes.h

```diff
@@ -10,22 +10,31 @@
 #include <winrt/Windows.Web.Http.h>
 #include <winrt/Windows.Web.Http.Filters.h>
 #include <winrt/Windows.UI.Core.h>
-#include <winrt/Windows.UI.Xaml.Automation.h>
-#include <winrt/Windows.UI.Xaml.Automation.Peers.h>
-#include <winrt/Windows.UI.Xaml.Controls.h>
-#include <winrt/Windows.UI.Xaml.Controls.Primitives.h>
-#include <winrt/Windows.UI.Xaml.Input.h>
-#include <winrt/Windows.UI.Xaml.Markup.h>
-#include <winrt/Windows.UI.Xaml.Media.h>
-#include <winrt/Windows.UI.Xaml.Shapes.h>
-#include <winrt/Windows.UI.Xaml.Documents.h>
-#include <winrt/Windows.UI.Xaml.Media.Imaging.h>
 #include <winrt/Windows.Data.Json.h>
 #include <winrt/Windows.Foundation.h>
 #include <winrt/Windows.Foundation.Collections.h>
 #include <winrt/Windows.Data.Xml.Dom.h>
 #include <winrt/AdaptiveCards.ObjectModel.Uwp.h>
+
 #include <winrt/Windows.UI.Text.h>
+#include <winrt/Windows.UI.Xaml.Automation.h>
+#include <winrt/Windows.UI.Xaml.Documents.h>
+#include <winrt/Windows.UI.Xaml.Input.h>
```

Comment:
you will definitely want to put all these includes in a PCH if you aren't using it yet, otherwise your compile times will be bad

Reply:
`CppWinRTIncludes.h` is only included in the PCH. That accomplishes the same thing right?

Reply:
@asklar we're following the same pattern `WinUI2` is using https://github.com/microsoft/microsoft-ui-xaml/blob/main/dev/inc/CppWinRTIncludes.h